mesh/mesh.go

```diff
@@ -623,3 +627,22 @@ func (msh *Mesh) GetRewardsBySmesherId(smesherID types.NodeID) ([]*types.Reward,
 func (msh *Mesh) LastVerified() types.LayerID {
 	return msh.trtl.LatestComplete()
 }
+
+type ErrMissingBlocks struct {
```

Comment:
NIT: the general convention in the stdlib is to name errors with the `ErrXXX` name but error types with `XXXError`. For instance the `fs` package has:

```go
var (
	ErrInvalid    = errInvalid()    // "invalid argument"
	ErrPermission = errPermission() // "permission denied"
	ErrExist      = errExist()      // "file already exists"
	ErrNotExist   = errNotExist()   // "file does not exist"
	ErrClosed     = errClosed()     // "file already closed"
)
```

And as type:

```go
type PathError struct {
	Op   string
	Path string
	Err  string
}
```

```suggestion
type MissingBlocksError struct {
```